server/controllers/index.ts

```diff
@@ -0,0 +1,3 @@
+import dummyController from './dummyController';
+
+export default dummyController;
```

Comment:
unneeded index

if you wanna use a router.ts to group everything thats the best use for such "index" files

Reply:
Do you mean that I don't have to use an index file in the controllers' directory, even if I use a ton of controllers and work with router files only?